.chloggen/kubeletAKS.yaml

```diff
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Should this be `enhancement` instead?

Reply:
Then maybe a `bug_fix` is more appropriate? Since we expect this change to fix kubeletstats receiver on AKS?